dashboard/routes/guild-manager.js

```diff
@@ -219,24 +219,44 @@ router.post("/:serverID/greeting", CheckAuth, async (req, res) => {
     Object.prototype.hasOwnProperty.call(data, "welcomeEnable") ||
     Object.prototype.hasOwnProperty.call(data, "welcomeUpdate")
   ) {
+    if (!data.content) {
```

Comment:
This logic can be simply merged below by simply deleting `data.content` in if
```js
if (data.content !== settings.welcome.content) {
      settings.welcome.content = data.content;
    }
```
Similarly others can be updated